server/middlewares/validators/videos/videos.ts

```diff
@@ -126,6 +129,92 @@ const videosAddValidator = getCommonVideoEditAttributes().concat([
   }
 ])
 
+const videosAddResumableValidator = getCommonVideoEditAttributes().concat([
+  body('name')
+    .trim()
+    .custom(isVideoNameValid)
+    .withMessage('Should have a valid name'),
+  body('channelId')
+    .customSanitizer(toIntOrNull)
+    .custom(isIdValid).withMessage('Should have correct video channel id'),
+
+  async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    const videoFileMetadata = {
+      mimetype: req.body.mimeType,
+      size: req.body.size,
+      originalname: req.body.name
+    } as express.FileUploadMetadata
+    const user = res.locals.oauth.token.User
+    const file: Express.Multer.File & { id: string, metadata: any } = req.body
```

Comment:
What is `id`? And is it possible to type `metadata`?

Reply:
`id` is the upload id, AFAIK. `metadata` has been typed to the interface provided by uploadx.